generated/nidcpower/library.py

```diff
@@ -263,22 +261,6 @@ def niDCPower_GetExtCalRecommendedInterval(self, vi, months):  # noqa: N802
                 self.niDCPower_GetExtCalRecommendedInterval_cfunc.restype = ViStatus  # noqa: F405
         return self.niDCPower_GetExtCalRecommendedInterval_cfunc(vi, months)
 
-    def niDCPower_GetLastExtCalLastDateAndTime(self, vi, month):  # noqa: N802
```

Comment:
I wonder if we should call all these functions "fancy whatever" so it's obvious they are only fancy python wrapping functions.

Reply:
In this PR?

Reply:
no way